feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/account/list/multipleSelecting/SelectMultipleWalletsViewModel.kt

```diff
@@ -66,7 +66,12 @@ class SelectMultipleWalletsViewModel(
 
     override fun accountClicked(accountModel: AccountUi) {
         val selected = mutableSetOf(*selectedMetaAccounts.value.ids.toTypedArray())
+
         if (selected.contains(accountModel.id)) {
+            if (selected.size <= request.min) {
+                return
+            }
```

Comment:
I suggest to disable the switch in UI as well to avoid confusing user with "i clicked but nothing happened' behavior

Reply:
As we discussed we will improve it in next releases